functions.lua

```diff
@@ -80,12 +80,6 @@ function travelnet.is_elevator(node_name)
 	return node_name == "travelnet:elevator"
 end
 
-function travelnet.is_travelnet_or_elevator(pos)
```

Comment:
Have to comment that I would like generic API functions like this but no use for it currently so wont be requesting changes, just saying.